src/Talk/Update/UpdateTalkRequest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Talk\Update;
+
+use App\Talk\Create\CreateTalkRequest;
+use App\Talk\Talk;
+
+/**
+ * @method static UpdateTalkRequest createFromEntity(Talk $talk)
```

Comment:
Please remove this, it comes from the parent class, so can be inferred and probably is by any good static analysis tool and IDE

Reply:
PHPStorm can't infer the return type correctly, that's why I had to add this. They may have finally solved this issue in the incoming release, as they posted about that 2 days ago : https://blog.jetbrains.com/phpstorm/2019/08/phpstorm-2019-2-2-preview/